godtoolsTests/App/Share/Data-DomainInterface/Supporting/GetTranslatedLanguageName/GetTranslatedLanguageNameTests.swift

```diff
@@ -6,203 +6,216 @@
-                    let translationFromLocale: String? = languageNames[LanguageCodeDomainModel.french.value]?[LanguageCodeDomainModel.arabic.value]
-                    
-                    let translation: String? = getTranslatedLanguageName.getLanguageName(
-                        language: frenchLanguage,
-                        translatedInLanguage: LanguageCodeDomainModel.arabic.rawValue
-                    )
-                    
-                    expect(translation).to(equal(fallbackName))
-                    expect(translationFromAppBundle).to(beNil())
-                    expect(translationFromLocale).to(beNil())
-                }
-            }
-        }
+        let translation: String? = Self.getTranslatedLanguageName.getLanguageName(
+            language: russianLanguage,
+            translatedInLanguage: translateInLanguage
+        )
+        
+        #expect(translation == russianFallbackName)
+    }
+    
+    @Test(
+        """
+        Given: User is viewing the russian language name.
+        When: Translating the russian language name in spanish and forceLanguageName is true and a translation is available in the app bundle's string phrases and in Locale.
+        Then: Translate the russian language name using the fallbackName.
+        """
+    )
+    func usesFallbackInsteadOfAppBundleAndLocale() {
+       
+        let russianFallbackName: String = "Russian Fallback Name"
+        
+        let russianLanguage = MockTranslatableLanguage(
+            languageCode: LanguageCodeDomainModel.russian.rawValue,
+            localeId: LanguageCodeDomainModel.russian.rawValue,
+            fallbackName: russianFallbackName,
+            forceLanguageName: true,
+            regionCode: nil,
+            scriptCode: nil
+        )
+        
+        let translation: String? = Self.getTranslatedLanguageName.getLanguageName(
+            language: russianLanguage,
+            translatedInLanguage: LanguageCodeDomainModel.spanish.rawValue
+        )
+        
+        let translationFromAppBundle: String? = Self.localizableStrings[LanguageCodeDomainModel.spanish.value]?[LanguageCodeDomainModel.russian.value]
+        let translationFromLocale: String? = Self.languageNames[LanguageCodeDomainModel.russian.value]?[LanguageCodeDomainModel.spanish.value]
+        
+        #expect(translation == russianFallbackName)
+        #expect(translationFromAppBundle == "Ruso")
+        #expect(translationFromLocale == "Ruso")
+    }
+    
+    @Test(
+        """
+        Given: User is viewing the french language name.
+        When: Translating the french language name in spanish and forceLanguageName is false and a translation is available in the app bundle's string phrases.
+        Then: Translate the french language name using the app bundle's string phrases.
+        """
+    )
+    func usesAppBundlesStringPhrases() {
```

Comment:
Hey @rachaelblue I'm starting to add tests for the Share/Data-DomainInterface/Supporting.  I transitioned the current tests from Quick and Nimble to Swift Tests.  What do you think of supplying Given, When, Then in the ```Test``` description?

One thing I do like about Quick and Nimble is we don't have to define a method.  In SwiftTests I have to define a method which can get wordy.

Reply:
Ahh yeah the comment would be a good place for additional clarification.